README.md

```diff
@@ -164,4 +167,4 @@ https://github.com/norwoodj/helm-docs
 
 `helm schema-gen values.yaml > values.schema.json`
 
-https://github.com/karuppiah7890/helm-schema-gen
\ No newline at end of file
+https://github.com/karuppiah7890/helm-schema-gen
```

Comment:
Ik weet niet waarom deze commands onderaan deze README staan?

Reply:
Oops, deze waren bedoelt als linkjes voor de documentatie / installatie van die tools. Horen dus in het tekstje zelf thuis